docs/pages/notifications.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Notifications
+description: Provides a detailed breakdown of Teleport's notification system.
+---
+
+Teleport's notification system allows users to be notified of various events, updates, and warnings in real-time.
+
+This guide explains how to interact with notifications in the Web UI and how cluster administrators can create custom notifications for their users.
+
+## Interacting with notifications in the Web UI
+
+In the Web UI, you can list all your notifications by clicking on the bell icon in the top right of the navigation bar.
+
+Clicking on a notification will redirect you to the relevant page, or in the case of a custom notification generated by an administrator, open a dialog containing its text content.
+You can mark the notification as read to acknowledge it, or hide it to have it never be shown to you again.
+
+Some notifications may include quick action buttons which allow you perform actions directly from the notification, such as assuming granted roles from an approved access request notification.
+
+![Notification in the WebUI](../img/notification.png)
+
+## Creating and managing notifications
+
+Cluster administrators can create and manage custom notifications using the `tctl notifications` command. These notifications contain custom text content and can be configured to target either all users, a specific user, or users with certain roles.
+
+Please note that custom notifications should not include any sensitive information, and should not be used for security-critical purposes.
+
+## tctl notifications create
```

Comment:
Instead of the command reference here, I would include examples of `tctl notifications` commands with brief descriptions of what they do. We can then move the `tctl notifications` sections into the `tctl` CLI reference so we have a comprehensive list of commands, and link to those sections in this guide.